source/display/objects/game/Note.hx

```diff
@@ -80,20 +80,13 @@ class Note extends FlxSprite
 				antialiasing = true;
 		}
 
+		x += swagWidth * noteData;
 		switch (noteData)
 		{
-			case 0:
-				x += swagWidth * 0;
-				animation.play('purpleScroll');
-			case 1:
-				x += swagWidth * 1;
-				animation.play('blueScroll');
-			case 2:
-				x += swagWidth * 2;
-				animation.play('greenScroll');
-			case 3:
-				x += swagWidth * 3;
-				animation.play('redScroll');
+			case 0: animation.play('purpleScroll');
+			case 1: animation.play('blueScroll');
+			case 2: animation.play('greenScroll');
+			case 3: animation.play('redScroll');
```

Comment:
[this code has already been refactored in my local changes](https://github.com/ThePlank/PlankEngine/assets/75984453/e85329f9-8fd2-4460-8ac5-4dd5ab561759)
also thank you for reminding me to actually commit my changes

Reply:
nice font